changelog.md

```diff
@@ -1,3 +1,26 @@
+1.9.0
+==========
+### New Features:
+**Microsoft Identity Web now exposes a token provider that the Azure SDKs can use**. See [PR](https://github.com/AzureAD/microsoft-identity-web/pull/542) for details.
+
+**Microsoft Identity Web now supports .NET Framework 4.6.2**. See issue [#1086](https://github.com/AzureAD/microsoft-identity-web/issues/1086).
+
+**Microsoft Identity Web supports calls for regional STS for 1st party only**, this is due to MSAL.NET release [4.29](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/releases/tag/4.29.0), and is available via the `ConfidentialClientApplicationOptions`.
+
+### Bug Fixes:
+**Microsoft Identity Web now locks on the HttpContext, to better handle multi-threaded applications**. See issue [#1097](https://github.com/AzureAD/microsoft-identity-web/issues/1097) and [PR](https://github.com/AzureAD/microsoft-identity-web/pull/1082) and [PR](https://github.com/AzureAD/microsoft-identity-web/pull/1099).
+
+### Fundamentals:
+**Microsoft Identity Web now implements `LoggerMessage` for high performance logging**. See issue [#1105](https://github.com/AzureAD/microsoft-identity-web/issues/1105) for details.
+
+**Performance improvements**. See PRs [#1089](https://github.com/AzureAD/microsoft-identity-web/pull/1089), [#1098](https://github.com/AzureAD/microsoft-identity-web/pull/1098), [#1092](https://github.com/AzureAD/microsoft-identity-web/pull/1092), and [#1085](https://github.com/AzureAD/microsoft-identity-web/pull/1085).
+
+### Documentation:
+**Documentation updated to show how to use `ClientCapabilities`**. See issue [#1071](https://github.com/AzureAD/microsoft-identity-web/issues/1071).
+
+**Clearly document what is available in Microsoft Identity Web and when to use MSAL.NET, Microsoft Identity Web, or both**. See issue [#1057](https://github.com/AzureAD/microsoft-identity-web/issues/1057) and [Is MSAL.NET right for me?](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Is-MSAL.NET-right-for-me%3F)
```

Comment:
Clear documentation is available about when to use MSAL.NET, Microsoft.Identity.Web or both ?